xcode/Subconscious/Shared/Components/Common/SubtextView.swift

```diff
@@ -81,16 +80,20 @@ struct SubtextView: View {
                     if (!renderable.entries.isEmpty) {
                         TranscludeListView(
                             entries: renderable.entries,
-                            onViewTransclude: { entry in
-                                onViewTransclude(entry.address)
-                            },
-                            onTranscludeLink: onTranscludeLink
+                            onLink: onLink
                         )
                     }
                 }
             }
         }
         .expandAlignedLeading()
+        .environment(\.openURL, OpenURLAction { url in
+            guard let link = url.toSubSlashlinkLink()?.toEntryLink() else {
+                return .systemAction
+            }
+            onLink(link)
+            return .handled
+        })
```

Comment:
Capture link taps at the Subtext level and expose as an `onLink` callback.

This code seemed too fiddly to expect consumers to implement everywhere, so I thought it made more sense to approach it this way.

URL is parsed to SubSlashlinkLink and then converted to an EntryLink. Subtext doesn't know the sphere context though, so when using, you'll probably have to rebase the link one level up (which we do).